src/main/java/io/confluent/connect/jdbc/TimestampIncreasingTableQuerier.java

```diff
@@ -92,33 +94,51 @@ protected void createPreparedStatement(Connection db) throws SQLException {
       //  timestamp 1235, id 22
       //  timestamp 1236, id 23
       // We should capture both id = 22 (an update) and id = 23 (a new row)
-      builder.append(" WHERE (\"");
+      builder.append(" WHERE (");
+      builder.append(quoteString);
```

Comment:
What I did in the past is have a utility function quoteIdentifier(String identifier) that returns a quoted string. 

I suspect it may and up more readable and less error-prone.